spyder/utils/tests/test_encoding.py

```diff
@@ -31,6 +32,8 @@ def test_is_text_file(tmpdir):
 def test_files_encodings(expected_encoding, text_file):
     with open(os.path.join(__location__, text_file), 'rb') as f:
         text = f.read()
+        if PY2:
+            text = str(text)
```

Comment:
No, I was wrong, this wasn't the problem with python2

The problem is trying to convert bytes to string without knowing the encoding, in python3 `str(some_bytes_with_no_uff8_encoding)` return an empty string but It fails in python2